model-optimizer/mo/utils/check_ie_bindings.py

```diff
@@ -46,15 +46,10 @@ def import_core_modules(silent: bool, path_to_module: str):
 
         print("\t- {}: \t{}".format("Inference Engine found in", os.path.dirname(openvino.__file__)))
         print("{}: \t{}".format("Inference Engine version", ie_version))
-        print("{}: \t    {}".format("Model Optimizer version", mo_version))
+        print("{}: \t{}".format("Model Optimizer version", mo_version))
 
         versions_mismatch = False
-        # MO and IE version have a small difference in the beginning of version because
-        # IE version also includes API version. For example:
-        #   Inference Engine version: 2.1.custom_HEAD_4c8eae0ee2d403f8f5ae15b2c9ad19cfa5a9e1f9
-        #   Model Optimizer version:      custom_HEAD_4c8eae0ee2d403f8f5ae15b2c9ad19cfa5a9e1f9
-        # So to match this versions we skip IE API version.
-        if not re.match(r"^([0-9]+).([0-9]+).{}$".format(mo_version), ie_version):
+        if mo_version != ie_version:
```

Comment:
Do we want to support both versions: old and a new one?

Reply:
Discussed offline. Closing.